WordPress/src/main/java/org/wordpress/android/ui/comments/CommentAdapter.java

```diff
@@ -157,25 +172,71 @@ void setOnSelectedItemsChangeListener(OnSelectedItemsChangeListener listener) {
         return holder;
     }
 
+    private String getFormattedTitle(CommentModel comment) {
```

Comment:
Hmm not sure about this one, this comment adapter is hard linked to the comment list. We could spend more time on separating the adapter and the view, but I don't think this should be done right now.